tests/transfers.rs

```diff
@@ -776,4 +779,30 @@ fn tapret_wlt_receiving_opret() {
         1000,
         None,
     );
+    assert_eq!(consignment.bundles.len(), 0);
```

Comment:
But this must be 3, isn't it?

Reply:
Yes, 0 is incorrect. The ["fixing" PR](https://github.com/RGB-WG/rgb-std/pull/272) has introduced this bug, as explained in https://github.com/RGB-WG/rgb-tests/pull/6#issuecomment-2376494270

Reply:
If this is a bug I believe it must be a failing assertion, so we do not merge this PR unless the bug is fixed

Reply:
Please check the message of this PR, it says:
> This is not meant to be merged, it's just here to push ahead the discussion in https://github.com/RGB-WG/rgb-tests/pull/6

Reply:
The purpose of this PR is showing the tests are deterministic and there is no such case of consignment changing the number of bundles based on a run executed on a different hour/day. If I change the assertion the test would fail and it wouldn't be possible to show that it deterministically returns `0` bundles at every run. I don't see any issue in having a PR with a different approach since there are no better alternatives. Do you see alternatives?